account_invoice_date_maturity/views/account_move_views.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="account.action_move_journal_line" model="ir.actions.act_window">
+        <field name="context">{'default_type': 'entry', 'search_default_misc_filter':1}</field>
```

Comment:
I'd emphasize that change in this way (setting value to `False`, and documenting original context):
```suggestion
        <!-- Overriden context: {'default_type': 'entry', 'search_default_misc_filter':1, 'view_no_maturity': True} -->
        <field name="context">{'default_type': 'entry', 'search_default_misc_filter':1, 'view_no_maturity': False}</field>
```